storage/innobase/xtrabackup/src/xtrabackup.cc

```diff
@@ -7855,6 +7865,48 @@ append_defaults_group(const char *group, const char *default_groups[],
 	ut_a(appended);
 }
 
+/**************************************************************************
+Prints a warning for every table that uses unsupported engine and
+hence will not be backed up. */
+static
+void
+xb_tables_compatibility_check()
+{
+	const char* query = "\
+SELECT                                   \
```

Comment:
Please format string literal similar to https://github.com/percona/percona-server/blob/5.6/storage/innobase/dict/dict0crea.cc#L1402